src/style.css

```
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  background-color: gray;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
```

```suggestion
  width: 200px;
```
and use box-sizing: border-box